frontend/src/screens/wallet/Receive.tsx

```diff
@@ -43,6 +43,7 @@ export default function Receive() {
             <CardContent className="flex flex-col items-center gap-6 pt-6">
               <div className="relative flex items-center justify-center">
                 <QRCode
+                  level="Q"
```

Comment:
Can you also add this to other instances as well where there's an image overlay, ex: in `frontend/src/screens/apps/AppCreated.tsx` etc.?

Reply:
Yeah, that + the avatar was too big. But it still has a negative effect on scanning, especially I think if the lightning address is shorter